chord_metadata_service/chord/ingest.py

```diff
@@ -23,6 +23,7 @@
 __all__ = [
     "METADATA_WORKFLOWS",
     "WORKFLOWS_PATH",
+    "IngestError",
     "ingest_resource",
```

Comment:
A bit of nitpicking for consistency:
Should these also be in capital letters with underscores between words?

Reply:
No, in this case, it is a list that just exposes all different variables and functions that can be imported from here. And they are of different types that follow their own style, e.g. `IngestError`  follows class style in python recommended by PEP8, `ingest_resource ` follows recommended function style.

Reply:
Got it!
Thank you -- approved